src/App.jsx

```diff
@@ -2,17 +2,20 @@ import { useState } from 'react';
 import './App.scss';
 
 export const App = () => {
-  const [count] = useState(0);
+  const useStateVariable = useState(0);
+  let count = useStateVariable[0];
+  const setCount = useStateVariable[1];
 
   const addOne = () => {
-    // write code here
+    setCount(count + 1);
   };
 
   const add100 = () => {
-    // write code here
+    count += 100;
+
+    setCount((count));
```

Comment:
It would be better to update the state based on the previous, you can use this approach that allows you to get the current value from callback 
https://react.dev/reference/react/useState#updating-state-based-on-the-previous-state

```suggestion
    count += 100;

    setCount(currentCount => currentCount += 100);
```